### DISCUSSION TITLE
TAS-IX filter

### DISCUSSION THREAD

**Discussion by bluelayer** [tnodir/fort#69] [Q&A] [answered]
Hi!

I see this for quite long time but seems dead... no?

![1](https://user-images.githubusercontent.com/3768436/235119443-4648107a-09e2-4e83-8206-42ce547631a6.jpg)

**Comment by tnodir** [comment, marked as answer]
I used the TAS-IX addresses in Uzbekistan to filter them from Internet addresses.

I can't test it atm, because `http://mrlg.tas-ix.uz/index.cgi` works in Uzbekistan only.

Why do you think that it's dead?

**Reply by bluelayer** [reply]
> I can't test it atm, because `http://mrlg.tas-ix.uz/index.cgi` **works in Uzbekistan only.**
> 
> Why do you think that it's dead?

look like you gave the answer =)
